build/bundle.js

```diff
@@ -1 +1,4671 @@
\ No newline at end of file
+!(function(e) {
```

Comment:
@tamdao Whats the reason to change this file? We will be using travis to build this.